packages/@okta/vuepress-site/code/index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: Landing
+title: API Reference
+---
+
+# Languages & SDKs
+Code and resources for your platform.
+
+## Mobile
+
+<Cards class="cols-4">
+  <Card href="/code/android/" :showHeaderIcon=true headerIcon="code-android" :showFooter=false>Android</Card>
+  <Card href="/code/ios/" :showHeaderIcon=true headerIcon="code-ios" :showFooter=false>iOS</Card>
+  <Card href="/code/react-native/" :showHeaderIcon=true headerIcon="code-react" :showFooter=false>React Native</Card>
+</Cards>
+
+## Front-end
```

Comment:
I know we're not suppose to comment on content, but just so that this doesn't get lost:

Since the term is stand-alone and not describing something, this should be: Front End